OMeetMgmt/manage_events.php

```diff
@@ -82,10 +82,12 @@ function name_to_get_qrcodes_link($event_id) {
   $proto = "http://";
 }
 // Make sure that the base path doesn't end in a /, this makes life easier when crafting the links
-$base_path_for_links = $proto . $_SERVER["SERVER_NAME"] . dirname(dirname($_SERVER["REQUEST_URI"]));
+$base_path_for_links = $proto . $_SERVER["SERVER_NAME"] .":". $_SERVER['REMOTE_PORT'] . dirname(dirname($_SERVER["REQUEST_URI"]));
 if (substr($base_path_for_links, -1) == "/") {
   $base_path_for_links = substr($base_path_for_links, 0, -1);
 }
+// Uncomment if testing with a server that doesn't work with above base_path

```

Comment:
Just curious how you were testing - were you running with a local server or something?
Likely the way I would do this would be to add an additional parameter to the $_GET array - server_path_for_links - or something like that - and if that is set it will be used, otherwise the default logic above is used.  This way the code is more generic and the behavior can be changed by adjusting the URL used for manage_events.
If you're happy adjusting that, that's fine, or I can merge this in "as-is" and I can quickly make the adjustment.

Reply:
Testing in a localhost PHP server